client-data/js/board.js

```diff
@@ -49,6 +49,8 @@ Tools.showMarker = true;
 Tools.showOtherCursors = true;
 Tools.showMyCursor = true;
 
+Tools.isIE = /MSIE|Trident/.test(window.navigator.userAgent);
```

Comment:
Doesn't this trigger for the latest chrome-based IE ?

Reply:
No Edge has a different UA that completely drops the MSIE monicker. Only IE<=10 had MSIE and IE 11 had Trident
`Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.74 Safari/537.36 Edg/79.0.309.43`

Reply:
Perfect :)